kubernetes/clickhouse/README.md

```
@@ -0,0 +1,57 @@
+# Clickhouse Chart
+This chart defines Clickhouse DB Cluter as Kubernetes objects and it depend on the Clickhouse Operator, which is available at https://docs.altinity.com/clickhouse-operator
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)